### PR TITLE
Add script and support for AWS EMR with atlas checks

### PR DESCRIPTION
**Pull request by atiannicelli**
### Description:
This patch adds a new script that can be used to execute atlas checks using AWS EMR. This script is very similar to the one used for EMR in atlas-generator. Some small additional changes were made to support execution on EMR.

### Potential Impact:
This patch bumps the atlas-generator and atlas version to the latest (this was needed to get it working). All the other changes should not affect normal atlas checks execution.

### Unit Test Approach:
I executed this script against a couple countries and watched that the EMR processes all kicked off correctly, finished correctly , and that final atlas checks results were properly copied to the AWS s3 output folder specified. I am doing a global execution now to make sure it all works and see how long it takes.

### Test Results:
All good.

### CODE REVIEW

**Review by Bentleysb — COMMENTED** [review]
This looks good to me. Can't wait to see how the global run goes.

**Review by Bentleysb — APPROVED** [review]

**Review by rheatreena — APPROVED** [review]

**Comment by Bentleysb** [comment]
@atiannicelli, can you please resolve the merge conflicts?

**Review by sayas01 — COMMENTED** [review]
LGTM! Will approve once the merge conflicts are resolved.

**Comment by atiannicelli** [comment]
@Bentleysb and @sayas01 I resolved the merge conflicts and added one more section that will add some EMR autoscaling for when we get down to only processing a few tasks that last for hours. This makes it so we don't have so many nodes running unless we need them.

**Review by Bentleysb — APPROVED** [review]

File: config/configuration.json
Comment (sayas01) [inline config/configuration.json:365]: nit: Looks like your other PR(https://github.com/osmlab/atlas-checks/pull/622) includes this change.

**Review by sayas01 — APPROVED** [review]
LGTM!

**Pull request merged**
